src/components/Inputs/BirthdateInput/BirthdateInput.js

```diff
@@ -66,16 +61,7 @@ const PrivateBirthdateInput = (props) => {
     }
   }
 
-  const doValidation = (value, isTouched) => {
-    // User hasn't blurred but we still need to inform form
-    // engine if we're in a valid state or not
-    if (!isTouched && !!formChangeHandler) {
-      callErrorHandlers(value, formChangeHandler)
-    } else {
-      // Have blurred
-      callErrorHandlers(value, setErrorWrapper)
-    }
-  }
+  const [doValidation] = useInputValidation({validate, setError, formChangeHandler, callErrorHandlers})
```

Comment:
Can't see from this  diff, but I have an override `callErrorHandlers` defined before this. I also added example in the `useInputValidation` top comment, but this allows us to define custom implementations of the methods if boiler-plate of say, just one of them, doesn't work for our case.